contributed_definitions/nyaml/NXresolution.yaml

```diff
@@ -0,0 +1,154 @@
+category: base
+doc: |
+  Describes the resolution of a physical quantity.
+type: group
+NXresolution(NXobject):
+  physical_quantity:
+    unit: NX_ANY
```

Comment:
NX_CHAR should not have an NX_UNIT_CATEGORY

Reply:
You are right, this was an old leftover. Has been fixed.